freeglut/freeglut/src/fg_internal.h

```diff
@@ -526,11 +579,14 @@ typedef void (*SFG_Proc)();
  * and for no other reason.  Since it's hidden in the macro, the
  * ugliness is felt to be rather benign.
  */
-#define SET_WCB(window,cbname,func)                            \
+#define SET_WCB(window,cbname,func,udata)                      \
 do                                                             \
 {                                                              \
     if( FETCH_WCB( window, cbname ) != (SFG_Proc)(func) )      \
```

Comment:
Its not a strong opinion, but i guess else if makes logical sense / keeps code easiest to understand